### ISSUE TITLE
doesn't do anything on windows

### ISSUE THREAD

**Issue by LeetCodes** [atom-compare-files/atom-compare-files#52]
i open the command pallette and type compare and click it like the gif, and nothing happens, right click the file list on the side and it doesn't have compare and then nothing happens when i press ctrl + alt + c

**Comment by BRaimbault** [comment]
OS - windows 10
atom - 1.18.0
compare-file - 0.81

I experience the same problem.
Plus "compare files" does not appear if I right click on the two files in the treeview.

I am not familiar with the atom environment for developers let me know if there is anything else I can do to help.

Many thanks,
Bruno

**Comment by ParitoshBh** [comment]
Could you confirm if the name of files being compared is the same? If you notice carefully in demo, the file names are the same! I haven't really dug into the code to confirm this but looks like plugin only works with identical file names.

**Comment by danbrennerarc** [comment]
@ParitoshBh ,

Is there anyway to compare files with different names so that versioning doesn't impact the use of this package?

**Comment by gorillrman** [comment]
Same problem compare is not available through right-click, :
Windows 7
ATOM 1.22.1 x64
Compare-files 0.8.1

Have tried renaming files to be the same, no difference.

**Comment by photis** [comment]
I can confirm, doesn't work for: 
- Mac OS 10.13.2 High Sierra. 
- Atom 1.23.1 x64. 
- Compare-files 0.81
- same files names in different folders
- right-click doesn't show compare item
- clicking menu item doesn't lead to any action

**Comment by noxd3v** [comment]
Doesn't work on Windows 10 either

Atom 1.22.x64

the rest is same as here

> Compare-files 0.81
> same files names in different folders
> right-click doesn't show compare item
> clicking menu item doesn't lead to any action

**Comment by lominart** [comment]
Same problem for:

- macOS High Sierra 10.13.2
- Atom 1.23.2 x64
- compare-files 0.8.1

If I click Ctrl+Cmd+C it says "Compare Files: Select the two files to compare..." at the bottom.
If I select two files and click Ctrl+Cmd+C nothing happens.
If I select two files and select Packages > Compare Files > Compare nothing happens.
Right-click doesn't show compare item.

**Comment by jb0hn** [comment]
Same here. 

- Ubuntu 16.04.3
- Atom 1.23.2 x64
- compare-files 0.8.1

**Comment by jotavejv** [comment]
Same problem:

Win 10
Atom 1.23.3 x64
Compare files 0.8.1

**Comment by bebop350** [comment]
Does not seem to work.
Arch Linux x64 (latest)
Atom (latest)
Compare files (latest)

**Comment by cjvreeman** [comment]
Having the same issue.  Installed compare-files package and getting the message at the bottom of the screen when selecting a file and hitting ctrl-alt-c ("Compare Files: Select the two files ...").  Just not getting anything after that.

Windows 7 Professional, Service Pack 1
Atom (1.24.0 x64)
Compare files (0.8.1)

**Comment by Perfect-Solutions-Web-Services** [comment]
KDE Neon 5.12
Atom 1.25.1 X64
Compare files (0.8.1) Is there any other?

**Comment by RichSiciliano** [comment]
KDE Neon 5.12
Atom 1.25.1 X64
Compare files (0.8.1) Is there any other?

Compare-Files is a waste of time. Split-Diff works and serves my purpose.

**Comment by gmansillo** [comment]
Same issue with Atom 1.25.1 on Ubuntu 17.10 :cry:

**Comment by bpmarkovich** [comment]
Windows 10
Atom 1.25.1 X64
I'm having the same issue, am I doing something wrong? I would really like one single output file.

**Comment by LeetCodes** [comment]
nope, not doing anything wrong, just expected behavior and no plans to fix this I guess... I just stopped using Atom and switched to Visual Studio Code, I feel like Atom was like an alpha of VSCode to test the waters and see if the idea was actually do-able, VSCode is everything Atom is and more, and now Microsoft controls both so..  My only recommendation is to just forget it's made by Microsoft and has Visual Studio in the name, grit your teeth and try VS Code
